packages/styles/scss/components/background-media/background-media.scss

```diff
@@ -0,0 +1,88 @@
+//
+// Copyright IBM Corp. 2021
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '../../globals/utils/aspect-ratio';
+
+@mixin background-media {
+  :host(#{$dds-prefix}-background-media) {
+
+    .#{$prefix}--background-media {
+        width: 100%;
+        height: 100%;
+    }
+      
+    .#{$prefix}--background-media-container {
+        position:relative;
+        height: 100%;
+        width: 100%;
+
+        @include carbon--breakpoint-down('md') {
+            position: flex;
+            flex-direction: column;
+        }
+    }
+
+    .#{$prefix}--background-media-gradient {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        top: 0;
+        left: 0;
+        z-index: -1;
+
+        &:after {
+            content:'';
+            position: absolute;
+            left: 0;
+            top: 0;
+            width: 100%;
+            height: 100%;
+        }
+
+        .#{$prefix}--background-media-item {
+            @include carbon--breakpoint-down('md') {
+                position: relative;
+                padding-top: aspect-ratio(4, 3);
+                width: 100%;
+                order: 2;
+            }
+        }
+    }
+
+    .#{$prefix}--background-media-image {
+      object-fit: cover;
+      height: 100%;
+      width: 100%;  
+
```

Comment:
Looks like there's some extra spacing that needs to be adjusted. Try running `yarn format` or run `git commit` to get some of the format and linting in place.